core/src/main/scala/org/broadinstitute/dsde/rawls/metrics/RawlsInstrumented.scala

```diff
@@ -37,26 +43,48 @@ trait RawlsInstrumented extends DefaultInstrumented {
     */
   @implicitNotFound(msg = "Cannot expand instances of type ${A}")
   protected sealed trait Expansion[A] {
-    def makeName(key: String, a: A) = s"$key.${a.toString}"
+    def makeName(key: Option[String], a: A) = s"${getKey(key)}${a.toString}"
```

Comment:
so i understand: is this because stackdriver goes looking for the text `thing.thing_id` in the metric?